README.md

```diff
@@ -1,56 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Covid19 Tracker Web app
 
+> A web app built with Covid-19-API that displays Covid 19 cases, deaths and recovery per country in the entire World
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, JavaScript
+- React
+- Redux
+- WebPack 
+- Covid19 API
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo](https://634b18b7735def69fb36e1ca--musical-sunburst-e30cc3.netlify.app/)
 
 
-## Getting Started
+## Video Demo 
+
+[Video Demo](https://drive.google.com/file/d/15cvM6YNoK8lCaV5BIxsvKP2Tg24AxtFp/view?usp=sharing)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+## ScreenShots
+### Desktop screen
+<img src="./Screenshots/Homepage1.png" alt="Rockets" width="500px"/>
+<img src="./Screenshots/Homepage2.png" alt="Missions" width="500px"/>
 
+### Mobile Screen
+<img src="./Screenshots/homepageMobile.png" alt="Profile" width="500px"/>
+<img src="./Screenshots/detailsMobile.png" alt="Profile" width="500px"/>
+
+
+## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Install node.js and npm.
+- Install Git.
+
 ### Setup
 
+- Clone the repository by running the following command in your terminal:
+    git clone https://github.com/kidde60/covid-19-updates
+    
+- Change your directory by by running the following command in your terminal:
+    cd space-hub
+
 ### Install
+- Install the dependencies by running `npm i` in your terminal
 
 ### Usage
 
-### Run tests
+- After all the dependencies are installed open the app by running npm start in your terminal
+
 
 ### Deployment
 
 
 
 ## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.